### ISSUE TITLE
Some DSO names are dubious

### ISSUE THREAD

**Issue by fmeynadier** [Stellarium/stellarium#11]
Hello,

I am a bit surprised by some of the names appearing in my favorite planetarium software for some deep sky objects... 

I think this tracks back to commit ee300ad15668aea39aea3b604bef1dabc5f70321

Impacted file :  nebulae/default/names.dat (and possibly translations of it ?)

The indicated source for the names is : the list of 'The 500 Best Deep Sky Objects' by Don Pensack; https://www.cloudynights.com/topic/472872-500-best-dso-list/ 

(later tagged "B500" once comments were allowed, i.e from commit 919221490a660da0559bb529291546176abfa850)

Although the list itself is fine and certainly useful, the "name" column contains mostly nicknames for which I don’t find any primary source. To my opinion they should not appear in Stellarium, which should stick to official identifiers (or more commonly accepted name sources). 

I would suggest to remove the lines with B500 source (I can submit a PR if agreed).

Anyway thanks for the good work on Stellarium !

**Comment by gzotti** [comment]
While the DSO names are still work in progress and I very much insisted that we at least track a reference for some "funnies" and not collect everything from everywhere, I agree that some names require better source reference or more common acceptance. However, common acceptance requires previous publication in widespread journals and software and will likely take 3-20 years, so this is circular argumentation of a tricky kind. E.g. you will likely not find the name "Pillars of Creation" before that Hubble image.
I suggest that you write to Don Pensack and ask whether he can give references in a V2 of his Excel sheet: are the names his own invention, published elsewhere (journal, book, citable sources), learned at a star party from a particular observer group, ...  
I am not sure whether we should add more complications like filtering the list by user-selected sources. If you absolutely dislike some names, you can delete them.

**Comment by fmeynadier** [comment]
Thanks for your answer. Sorry if I missed some discussion on the subject (I did try to browse through the mailing list’s archive but did not find any discussion on that. I’d be happy to read any thread you would point me to).

Before writing to Don Pensack I’d prefer to know if anybody informed him already that his list has been incorporated in Stellarium, which may change a lot the content of my message.

But apart from that : I think a good rule of thumb would be to see if image search engines returns any relevant results. For example, "lawnmower cluster" (=NGC 663 according to B500, which is the one that initially raised my eyebrows) returns zero astronomical results, whereas "pillars of creation" returns nearly 100% NGC 6611/M16 - related results. "Eagle nebula" (which is the label I see currently in that region) is also OK according to this criterion.

**Comment by gzotti** [comment]
This was a private discussion between Alexander and myself. 
The "Pillars" were just an example of fast adoption of a new name in addition to the age-old (19th century?) name of Eagle Nebula, when it has been introduced in a picture published by a high-impact institution. (Actually it is pars-pro-toto, a small part of the Eagle nebula suddenly taking over the well-known name of M16.) I agree I have never seen the "Lawnmower" elsewhere and would like to see a primary reference on that and certainly a few others.

**Comment by fmeynadier** [comment]
OK. I was also aware that "Pillars" is a particular zone of M16, given its high notoriety I wouldn’t mind seeing its location pinpointed in Stellarium but it should not replace Eagle nebula or M16 (would feel like seeing "Eiffel Tower" instead of "Paris" on google earth !).

Concerning B500, honestly, lots of names look weird. Those are the first 10 occurences in names.dat : 

NGC  40              _("Scarab Nebula") <-- is better known as "Bow-Tie nebula", in fact this other name appears first in the names.dat list and is displayed
NGC  188             _("Polarissima Cluster") <-- http://www.messier.seds.org/m/m092.html suggests this name, but in 14000 years
NGC  206             _("Great Star Cloud in Andromeda Galaxy") <-- More of a description than a name 
NGC  246             _("Soap Bubble Nebula") <-- First hit directs to a distinct object. Wikipedia for NGC 246 mentions "Skull nebula" citing "Astronomy Now" magazine as a reference. Also cites "Pac Man" nebula as a nickname.
NGC  246             _("Voodoo Mask Nebula")  <--- Duplicate of the above within B500 list, no idea where this comes from
NGC  488             _("Whirligig Galaxy") <-- 1 single hit from http://www.cxielo.ch/gallery/f/ngc488, perhaps not independent from B500
NGC  584             _("Little Spindle Galaxy") <-- Lots of hits... on NGC 5866 aka "spindle galaxy". Besides I fail to see any resemblance between both objects, but that’s not the point...
NGC  628             _("Phantom Galaxy") <-- OK
NGC  654             _("Fuzzy Butterfly Cluster") <-- Butterfly cluster is NGC 6405 / M 6
NGC  663             _("Lawnmower Cluster") <-- Already discussed

So it appears to me that in 9 other 10 cases it is more a "remarks" column or mnemonics for the author’s use, but not identifiers that should appear very prominently in Stellarium.

**Comment by fmeynadier** [comment]
Ok, I got an answer from Don Pensack. Quoting his mail, "The common names in my list are from other amateurs, from Sue French and Steve O'Meara and other books I have. Each name is something I have heard or read." 
Quite understandably, he doesn’t have time to provide a source for each source. 

In the rest of his mail he makes a good point in explaining that giving names makes memorization easier for amateur astronomers wishing to keep track of the objects.  

My conclusion is that this list is interesting and is a nice addition to Stellarium, as it highlights DSOs that are reachable by (advanced) amateur astronomers (an equivalent list for southern hemisphere may be found at 
http://www.iceinspace.com.au/forum/showthread.php?t=45678, apparently). 

But I think it should be displayed as such (perhaps this is what you mean by "filtering the list by user-selected sources" ?), e.g. "The 500 Best Deep Sky Objects' by Don Pensack", with an identifier  like "B500 -#NN : Hairy Eyebrow Galaxy", not displayed by default.

And I strongly feel that, besides of that, any addition to the default "names.dat" should be at least checked against a basic search on google/bing/whatever, because this is exactly what users will do when they stumble on funny names on the sky map… If no results are astronomy-related, this might cast doubts on Stellarium’s accuracy.

**Comment by alex-w** [comment]
I've revisiting the names of DSO from various books and works going is not very fast (no google within books :) ), plus I added new option to the GUI to hide an additional names for DSO.

**Comment by alex-w** [comment]
Please check version 0.17.0 (I'll revisiting the names of DSO for next versions also)

**Comment by silas1037** [comment]
I am now checking the nickname of DSO and first find:

>I call M22 the Crackerjack Cluster (after the sweet popcorn treat with the prize in every box) because, at high power, after you penetrate the outer swarms of stars, many surprises await your gaze.

*Deep-sky companions: the Messier objects by Stephen James OMeara*

Here is the nickname "Crackerjack" of M22, which using a trademark. I don't support adding the same things into our sky, which will be mentioned by our later people, just like the Robur Carolinum.

@alex-w @gzotti

**Comment by gzotti** [comment]
Hmm, yes this family of topics is still open. We still need some way of selection/excluding unwanted sources for star names, DSO names, and probably also asterisms. A new Dialog in the Skyculture tab which loads the reference.fab and let the users select which references to take names from. And names only found in the unselected sources must then be suppressed.

"Crackerjack" may be something US citizens would probably know (if it is as widespread as OMeara seems to tell us) and refer to in everyday language. But it says nothing to me. I mentally translate this into "some US American [probably crispy] cookie", without even bothering whether it's sweet or salty. The connection with some prize is therefore lost, making it indeed a totally useless name. And yes, we should not introduce trademarks or product names into the sky :-)

**Comment by silas1037** [comment]
M86's name "FAUST V051" should be removed.
As mentioned [here](http://cdsannotations.u-strasbg.fr/annotations/simbadObject/1934764), "FAUST V051" is only a catalog number of [FAUST Observations of Ultraviolet Sources toward the Virgo Cluster](https://iopscience.iop.org/article/10.1086/313007) (pub 1997), we should not list it into the important Messier Nicknames.

**Comment by gzotti** [comment]
> M86's name "FAUST V051" should be removed. As mentioned at [simbad annotation](http://cdsannotations.u-strasbg.fr/annotations/simbadObject/1934764), "FAUST V051" is only a catalog number of a unknown survey [FAUST Observations of Ultraviolet Sources toward the Virgo Cluster](https://iopscience.iop.org/article/10.1086/313007) (pub 1997), we should not list it into the important Messier Nicknames.

Right, this is just "yet another catalog number" from a catalog where we don't have all objects. And it does have a memorizable name...

**Comment by silas1037** [comment]
M25 == υ Sgr Cluster ?
1. M25 is far from υ Sgr. I don't know why it calls υ Sgr Cluster.
2. I check *The Photographic Atlas of the Stars* as below,  and find it was full of errors, such as "6520 in M20" and "6530 in M20" :-( !
So maybe it should be removed or corrected.
![image](https://user-images.githubusercontent.com/36977717/153026503-0eb82807-f53f-47c6-8fd8-b213b43c127a.png)

**Comment by gzotti** [comment]
This &upsilon; Sgr Cluster entry is weird. Must be a misprint for &mu; Sgr Cluster.

OK, there are many errors here. Can you maybe collect all and send a pull request? If this book is too bad, we should either remove its entries (it it does not provide anything relevant), or at least throw out obvious errors.

**Comment by gzotti** [comment]
The point about referencing is that we (should) let users decide which names they want to activate, be it even a "suspicious" list like B500. If you can fill in the gaps (missing references) for some DSO names, this would also be helpful. E.g. why is a whale galaxy at the same time a herring galaxy? Or see the weird entries for IC3568... who (mis-)copied lime and lemon here?

**Comment by fmeynadier** [comment]
Thank you very much for having kept this in mind and finally fixed it ! I have tested the latest snapshot and think the way out you found is good. Well done !
